### PR TITLE
[15.0][MIG] web_sheet_full_width

### PR DESCRIPTION
**Pull request by JoanMForgeFlow**
Standard migration to version 15.0
@BernatPForgeFlow 
@LoisRForgeFlow 
@AaronHForgeFlow

### CODE REVIEW

**Review by BernatPForgeFlow — APPROVED** [review]
LGTM!

**Review by mariadforgeflow — APPROVED** [review]
LGTM:)

File: web_sheet_full_width/README.rst [hunk 1]
```diff
@@ -0,0 +1,88 @@
...
+
+|badge1| |badge2| |badge3| |badge4| |badge5| 
+
+This module was written to extend the functionality of the web client
+to get full width in the form view sheet.
+
+This module works in community edition and in enterprise edition.
+
+This module feature and many other features and improvements to the web backend
+are included in `web_responsive module <https://github.com/OCA/web/tree/13.0/web_responsive>`_.
```
Comment (AaronHForgeFlow) [inline web_sheet_full_width/README.rst:34]:
```suggestion
are included in `web_responsive module <https://github.com/OCA/web/tree/15.0/web_responsive>`_.
```

**Review by AaronHForgeFlow — APPROVED** [review]
Functional review

**Comment by LoisRForgeFlow** [comment]
@OCA/web-maintainers Could you merge?

**Comment by legalsylvain** [comment]
/ocabot merge patch
(No review)

**Pull request merged**